### PR TITLE
Tools Folder unit tests

### PR DESCRIPTION
**Pull request by georgiyazovaliiski**
Signed-off-by: Georgi Yazovaliyski <georgi.yazovaliiski@gmail.com>

**Changes**
1. Unit tests for hex.go functions
1. Unit tests for maphelper.go functions
1. Unit tests for validator.go functions

### CODE REVIEW

File: hedera-mirror-rosetta/tools/validator/validator_test.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	}
+
+	var expectedNil *types.Error = nil
+	expectedError := errors.Errors[errors.InvalidOperationsTotalAmount]
+
+	// when:
+	result := ValidateOperationsSum(testData)
+
+	// then:
+	assert.Equal(t, expectedNil, result)
```
Comment (Daniel-K-Ivanov) [inline hedera-mirror-rosetta/tools/validator/validator_test.go:27]: Maybe you can use directly `nil` here, instead of creating it as variable

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Review by failfmi — APPROVED** [review]

**Review by Daniel-K-Ivanov — APPROVED** [review]

File: hedera-mirror-rosetta/tools/maphelper/maphelper_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	// when:
+	result := GetErrorValuesFromStringErrorMap(testData)
+
+	// then:
+	assert.Equal(t, 1, len(result))
+	assert.Equal(t, error, result[0])
+}
+
+func newErrorDummy(code int32, retryable bool) *types.Error {
+	return errors.New(fmt.Sprintf("error_dummy_%d", code), code, retryable)
```
Comment (Perseverance) [inline hedera-mirror-rosetta/tools/maphelper/maphelper_test.go:42]: I think you can use here `Errorf` to achieve the same thing - https://golang.org/pkg/fmt/#Errorf

**Review by Perseverance — CHANGES_REQUESTED** [review]

File: hedera-mirror-rosetta/tools/maphelper/maphelper_test.go (hunk 2 above)
Comment (georgiyazovaliiski) [reply hedera-mirror-rosetta/tools/maphelper/maphelper_test.go:42]: Errorf returns an error type. I am using Sprintf, which returns a string, to format the name of the custom error type from the types package.

**Review by Daniel-K-Ivanov — APPROVED** [review]

File: hedera-mirror-rosetta/tools/maphelper/maphelper_test.go (hunk 2 above)
Comment (Perseverance) [reply hedera-mirror-rosetta/tools/maphelper/maphelper_test.go:42]: Got it you are right!

**Review by Perseverance — APPROVED** [review]

**Pull request merged**